ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -4554,7 +4561,6 @@ static bool ggml_backend_cuda_device_supports_op(ggml_backend_dev_t dev, const g
         case GGML_OP_COS:
         case GGML_OP_CLAMP:
         case GGML_OP_LOG:
-            return true;
```

Comment:
Did you intend to remove this `return true;` here?
Because it makes the switch fall-through the `GGML_OP_SSM_SCAN` implementation check, which is most probably not what you intended.